pyproject.toml

```
@@ -196,6 +196,8 @@ filterwarnings = [
     "ignore:Python 3.14 will:DeprecationWarning",
     # https://github.com/lancedb/lancedb/issues/2315
     "ignore:Support for class-based `config`:pydantic.warnings.PydanticDeprecatedSince20",
+    # https://github.com/pymupdf/PyMuPDF/issues/4392
+    "ignore::DeprecationWarning",
```

Could you try 

```suggestion
    "ignore:builtin type \w+ has no __module__ attribute:DeprecationWarning",
```

following the warnings listed in swig/swig@49a7d2b0c6aed00bd07a9a28569020d560c2658c

---

FYI @pmeier we need to escape the slash:

```
"ignore:builtin type \\w+ has no __module__ attribute:DeprecationWarning"
```

Made the change.